shex/CMUMPS_clinical_chemistry.shex

```diff
@@ -5,27 +5,27 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 PREFIX fm: <http://hokukahu.com/map/>
 BASE <http://hokukahu.com/shapes/>
 
-start = <labs>
+start = @<labs>
 
 <labs> {                                                                                 # <63-1003681>
   ^cmumps:clinical_chemistry-63 {                                                           # <63_04-6929184.919299_1003681>
-    a [cmumps:Lab_Result-63],
-  # cmumps:identifier                     LITERAL  %map:{ fm:order.identifier %},          # "63-000007"
-    rdfs:label                          LITERAL,                                         # "DOEXXX,JANE"
+    a [cmumps:Lab_Result-63];
+  # cmumps:identifier                     LITERAL  %map:{ fm:order.identifier %};          # "63-000007"
+    rdfs:label                          LITERAL;                                         # "DOEXXX;JANE"

```

Comment:
Is this correct, with a semi-colon?  "DOEXXX;JANE"
Looks like it might be a search-and-replace error.